### PR TITLE
Minor fixes to README.md

### PR DESCRIPTION
**Pull request by wb8tyw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,10 @@ Each entry has the following members:
 An optional password member is if a password is needed.  The password
 should not be stored in this file.
 
-Advanced maintainers can copy the tests/pre-commit script to .git/hooks which if the proper tools are installed, will test the format of the files in a
-git commit action.  This requires shellcheck, yamllint, and codespell to be installed.
+Advanced maintainers can copy the tests/pre-commit script to .git/hooks which
```
Comment (dslotter) [inline README.md]: Advise changing "to .git/hooks" to "inside of .git/hooks directory"

Comment (wb8tyw) [reply README.md]: Updated in the current push

**Comment by maurizioandreotti** [comment]
Hi, what is the meaning of putting Active as true false? I guess this originates from the d-rats config options?

 Is it to track in the list also ratflectors not anymore active? in the case maybe we should also ask a date of when the line was added or last updated?  

or is it to populate the drats config flag? (ps. If I recall well, users should not config more than one ratflector at a time as otherwise loops of messages gets created between ratflectors, at least in d-rats<0.3.10... unless loops are resolved now or d-rats forces now only one ratflector to be in use)

if we go with additional fields for dates ect, if possible i would also suggest to have an explicit field for the contact of the ratflector maintainer so to be able to to check it's status in the future and avoid relying on word of mouth whenever is possible

**Comment by wb8tyw** [comment]
The "Active" as true/false is for cases where it is known that the Ratflector is temporarily offline for some reason.

Adding a field for the maintainer contact info is a good idea.  I do not have that information right now, and we would be dependent on the community to maintain that information.  We need to be careful about putting real e-mail addresses in the data as if the un-mangled e-mail address is public, so we might want to recommend maintainer#ratflector.example.com format.

We can add what ever fields that users want to maintain in the format.   I am not sure if I can get D-Rats to easily display all the extra informational fields, so I documented a multi-line description YAML format that allows putting in any text data.
It may be easiest for maintenance of D-Rats to just tell people to put any additional information there.

Potential looping would be different tickets for D-Rats if we do not already have them.

Normal messages in D-Rats clients only go to one "Radio" at a time, so do not cause a loop.
QSTs can go to multiple "Radios" at a time, but generally do not request a response, so I do not see them causing a loop.

I think only a ratflector can be configured with mutiple ratflectors in it, and I supposed that can cause a loop, if multiple ratflectors are configured that way.  Not sure how to stop that with the current protocol, except to caution to be very careful about ratflector configuration.

There was also appears to be an attempt at loop prevention in the existing D-Rats client that causes D-Rats client to simply lock up on some operations where it attempts to talk to it self, that I have not fully identified.  It simply drops some packets from itself to itself, which breaks the protocol and causes d-rats to go into an infinite loop of re-trying the dropped packets.  The dropping in addition to the denial of service removes an important tool for users to troubleshoot their own d-rats client by it self.

We know we have an issue with collision detection on automated responses to radio based repeaters where transmissions will be lost or cause QRM that we need to figure out a fix for.

**Review by maurizioandreotti — APPROVED** [review]
ok

**Pull request merged**